src/test/java/com/revature/servicestests/SearchServiceTest.java

```diff
@@ -368,8 +373,10 @@ void searchServiceBadModule()
 		String format = content.getFormat();
 
 		List<Integer> mlist = new ArrayList<Integer>();
+		List<String> tempArr = new ArrayList<String>();
 		mlist.add(module3.getId());
-		Set<Content >filtered = ss.filter(title, format, mlist);
+		//temparr IS TEMPORARY TO SILENCE ERRORS
```

Comment:
delete comment

Reply:
Comment necessary to indicate purpose of temporary variable